limitador-server/kubernetes/README.md

```diff
@@ -0,0 +1,220 @@
+<!-- omit in toc -->
+# Kubernetes
+
+The purpose of this documentation is to deploy a sample application published via AWS ELB, that will be ratelimited at infrastructure level, thanks to the use the envoyproxy sidecar container, that will be in charge of contacting to a  ratelimit service (limitador), that will allow the request (or not) if it is within the permitted  limits.
+
+There are mainly two recommended way of using limitador in kubernetes:
+1. There is an ingress based on envoyproxy that contacts with limitador ratelimit service before forwarding (or not) the request to the application
+1. There is an envoyproxy sidecar container living in the application pod that contacts with limitador ratelimit service before forwarding (or not) the request to the main application container in the same pod
+
+In this example it will be described the second scenario (where there is an application with an envoyproxy sidecar container contacting to limitador service).
+
+> **NOTE**
+><br /> If you don't want to manually manage the sidecar container definitions on your deployments (harcoding the container spec, loading the envoy configuration from a configmap that requires a pod restart to reload possibles configuration changes...), you can use marin3r, a light weight envoy control plane that allows you to inject envoyproxy sidecar containers and dynamically consume configs from Kubernetes custom resources.
```

Comment:
It'd be good to have a link to the marin3r repo here.

Reply:
totally true! I just removed from a previous doc version, I have just pushed it again